scripts/sysman/temperature.yml

```diff
@@ -21,12 +21,18 @@ etors:
       desc: "The maximum temperature across all sensors in the GPU"
     - name: MEMORY
       desc: "The maximum temperature across all sensors in the local memory"
+    - name: GPU_BOARD
+      desc: "The maximum temperature across all sensors in the GPU Board"
+      version: "1.7"  
     - name: GLOBAL_MIN
       desc: "The minimum temperature across all device sensors"
     - name: GPU_MIN
       desc: "The minimum temperature across all sensors in the GPU"
     - name: MEMORY_MIN
       desc: "The minimum temperature across all sensors in the local device memory"
+    - name: GPU_BOARD_MIN
+      desc: "The minimum temperature across all sensors in the GPU Board"
```

Comment:
IS MIN temperature really required ? Earlier we removed this support, MAX temp is interesting telemetry to aware, is the min temp requirement ?

Reply:
for now the spec changes here maintains the symmetry (to have MAX and MIN values). implementation can choose to expose only MAX right?

Reply:
There are 2 board sensors - one will be MIN and one will be MAX